test/ioc/IocTest.java

```diff
@@ -0,0 +1,49 @@
+package ioc;
+
+
+import exceptions.ioc.IocDuplicatedKeyException;
+import exceptions.ioc.IocKeyNotFoundException;
+import exceptions.ioc.IocKeyNullPointerException;
+import exceptions.ioc.IocValueNullPointerException;
+import org.junit.Before;
+import org.junit.Test;
+
+public class IocTest {
+  private Ioc ioc = null;
+
+  @Before
+  public void setup() {
+    ioc = Ioc.getInstance();
+    ioc.register("Testing", "Hello");
```

Comment:
Please create a test utility where you pull these values from instead of harcoding  the values everytime